components/merkledb/src/views/tests.rs

```diff
@@ -1000,7 +1001,7 @@ fn valid_name_for_url() {
 }
 
 #[test]
-#[should_panic(expected = "Wrong characters using in name")]
+#[should_panic(expected = "InvalidCharsInName")]
```

Comment:
Not the best way to report panics to users. It could make sense to replace

```rust
SomeIndex::from_access(self, addr.into()).unwrap()
```

in `AccessExt` with something more readable, e.g.

```rust
SomeIndex::from_access(self, addr.into()).unwrap_or_else(|e| {
    panic!("MerkleDB error: {}", e);
})
```

Could be done in a separate PR, I guess.

Reply:
Sure, will do.